src/config/link.rs

```diff
@@ -12,6 +12,9 @@ pub enum LinkType {
     Script,
     Cert,
     Key,
+    ReflectionWithFileName,
```

Comment:
```suggestion
    GrpcReflection,
```
just `Reflection` is too general term